clusterloader2/testing/load/experimental/extended_config.yaml

```diff
@@ -0,0 +1,313 @@
+# ASSUMPTIONS:
```

Comment:
What is the difference between this and the other one that you moved?

Reply:
This is the exact copy of the current load/config.yaml. Because we cannot have services and deployments bundled together I'm starting with a fresh load/config.yaml. I'll be adding new objects here, starting with secrets and configmaps